pkg/collector/tcp.go

```diff
@@ -40,6 +41,43 @@ func (cp *collectingProcess) startTCPServer() {
 	}
 }
 
+func (cp *collectingProcess) StartTLSServer(serverCert []byte, serverKey []byte) {
```

Comment:
Is TLSserver not supported with UDP transport? I see TLSclient in exporting process written for both TCP and UDP transport.

Reply:
Just realized you had WIP in the title. If you are planning to address the above already, please ignore the comment.

Reply:
For UDP Transport, we will need to use DTLS. Still working on that.

Reply:
StartTLSServer(serverCert []byte, serverKey []byte) can be simplified as StartTLSServer(serverCert, serverKey []byte)

Reply:
@shihhaoli Thanks! will update it.